deployment/terraform/efs.go

```diff
@@ -0,0 +1,87 @@
+package terraform
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/mattermost/mattermost-load-test-ng/deployment/terraform/ssh"
+	"github.com/mattermost/mattermost/server/public/shared/mlog"
+)
+
+const (
+	efsDirectory  = "/mnt/mattermost-data"
+	svcScript     = "/opt/mount.sh"
```

Comment:
Nit: what does `svc` stand for? Service? Maybe we can just call it `script`?

Reply:
Wouldn't be it better to keep it more understandable across the package? Maybe even `efsScript` or something like that?